### PR TITLE
Add Coveralls reporting

### PR DESCRIPTION
**Pull request by suda**

### CODE REVIEW

**Comment by dmiddlecamp** [comment]
seems pretty safe?

**Comment by suda** [comment]
Yeah, it doesn't touch how whole CLI works

**Comment by towynlin** [comment]
Just checked out the branch and ran all the same commands travis does, including `npm run coveralls` :+1: Thank you @suda !!

File: .travis.yml [hunk 1]
```diff
@@ -5,8 +5,13 @@ node_js:
 - '4'
 sudo: false
 script:
-- npm run lint
-- npm test
+  - npm run lint
+  - npm test
+matrix:
+  allow_failures:
+    - node_js: "4"
```
Comment (brycekahle) [inline .travis.yml]: This is old, we no longer want to allow failures on node 4.

**Pull request merged**